src/plugins/cal-events/cal-events.js

```diff
@@ -80,22 +79,31 @@ var componentName = "wb-calevt",
 	},
 
 	processEvents = function( $elm ) {
-		var settings = $.extend( {}, window[ componentName ], $elm.data( dataAttr ) ),
-			year, month, events, minDate, containerId, $container;
+		var currDate = new Date(),
+			currYear = currDate.getFullYear(),
+			currMonth = currDate.getMonth(),
+			events, minDate, maxDate, containerId, $container;
 
 		events = getEvents( $elm );
 		containerId = $elm.data( "calevtSrc" );
 		$container = $( "#" + containerId ).addClass( componentName + "-cal" );
 
 		minDate = events.minDate;
-		year = settings.year || minDate.getFullYear();
-		month = settings.month || minDate.getMonth();
```

Comment:
`settings.year` and `settings.month` need to be preserved because they are overrides. This needs to be rewritten in a way that the default settings are only used if they are not present in the settings object